src/main/java/org/mule/extension/http/api/request/builder/HttpRequesterRequestBuilder.java

```diff
@@ -115,7 +116,7 @@ public String replaceUriParams(String path) {
         throw new NullPointerException(format("Expression {%s} evaluated to null.", uriParamName));
       }
 
-      path = path.replaceAll("\\{" + uriParamName + "\\}", uriParamValue);
+      path = path.replaceAll("\\{" + uriParamName + "}", quoteReplacement(uriParamValue));
```

Comment:
I think you deleted extra `\\`

Reply:
Only the opening brace needs to be scaped